sourcegraph/learn
# SourcegraphSearch input styles update

### What should this PR do?
Resolves [DEVED-112](https://sourcegraph.atlassian.net/browse/DEVED-112) by refactoring styles on SourcegraphSearch input fields.

Screenshots below.

### Why are we making this change?
Addresses issues that came up in #162 including longer inputs for search.

### What are the acceptance criteria? 
- Styles look reasonable on desktop and mobile.

### How should this PR be tested?
- Check out branch, and take a look at the search inputs on structural-search-patterns, which has longer queries.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
